.babelrc

```diff
@@ -0,0 +1,13 @@
+
+{
+  "presets": ["es2015-node5"],
```

Comment:
Ah, great point.  I feel like libraries should be more generic, but in this case, I guess it's ok.

It kind of makes me want to publish raw ES6 source to npm, and let the application build step be responsible for making that ES6 source consumable.  Maybe we should revisit that (but not hold up merging this PR)?